src/Cli/dotnet/commands/dotnet-vstest/Program.cs

```diff
@@ -1,12 +1,12 @@
 ﻿// Copyright(c) .NET Foundation and contributors.All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.DotNet.Cli.Utils;
 using Microsoft.DotNet.Cli;
 using System.CommandLine.Parsing;
 using System.Collections.Generic;
 using System.Linq;
 using System;
+using Microsoft.DotNet.Tools.Test;
```

Comment:
Looks like MS namespaces are defined before `System` here, could you please move this using?